docs/set_up_svs.rst

```diff
@@ -21,6 +21,9 @@ epoxy to physically disable it. If you have questions about repurposing
 hardware for the *Secure Viewing Station*, contact the `Freedom of the
 Press Foundation <https://securedrop.org/help>`__.
 
+We also recommend you remove all speakers from the device, instead using
+earphones to listen to audio submissions.
```

Comment:
I think this is fine to include, ideally with a link to a blog/article/paper that explains to admins what the concern is. Hopefully as the admins are taking apart the SVS anyway to pull out other hardware then doing this as well is not too difficult (with that said, the list of modifications to the SVS is becoming _quite_ onerous...)

This flows best if it is moved to after the sentence ending in "physically disable it.".